e2e/tests/plugins/imagery/exampleImagery.e2e.spec.js

```diff
@@ -181,8 +181,11 @@ test.describe('Example Imagery Object', () => {
 
     });
 
-    test('Can use the reset button to reset the image', async ({ page }, testInfo) => {
-        test.slow(testInfo.project === 'chrome-beta', "This test is slow in chrome-beta");
+    test('Can use the reset button to reset the image', async ({ page, browser }, testInfo) => {
+        test.slow(
```

Comment:
Would you mind running this locally and specifying the chrome-canary config? You'll have to do a `npx playwright install chrome-canary` first

Reply:
I'm unable to get this to work. `npx playwright install chrome-canary` is not valid, and Playwright fails to find canary google chrome on my Mac (even after force reinstalling), and it's unavailable in linux. Were you able to get this working locally at some point in the past?